newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -63,6 +65,13 @@ export default class LocalPreviewLauncher extends React.Component<
 
     const win = new BrowserWindow(this.state.previewBrowserWindowConfig);
     win.loadURL(`file://${this.state.previewGamePath}/index.html`);
+    win.webContents.on('devtools-opened', () => {
```

Comment:
Nice synchronisation between React state and the devtools 👍

Reply:
thanks, this was driving me nuts when doing the extension last time :D